dependent/european-journal-of-epidemiology.csl

```diff
@@ -11,7 +11,7 @@
     <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
     <category citation-format="numeric"/>
     <category field="medicine"/>
-    <issn>0393-2990</issn>
+    <issn>1573-7284</issn>
     <eissn>1573-7284</eissn>
```

Comment:
That is another problem Travis is complaining about: The issn and eissn is the same. Actually the one which is getting deleted is correct.